Automattic/wp-calypso
# HelpCenter: fix iframe width and height for videopress

The iframe for videopress embeds was displaying with the initial width and height.
It wasn't customized based on the width of the help center container, and this PR aims to fix that

| Before  | After |
| ------------- | ------------- |
|  <img width="412" alt="Screenshot 2024-08-26 at 14 29 22" src="https://github.com/user-attachments/assets/1607701f-16fc-4748-b24f-0e4453ed029b"> | <img width="411" alt="Screenshot 2024-08-26 at 14 28 29" src="https://github.com/user-attachments/assets/651d388d-56af-4f62-b1e9-8d19e30d90ff">  |


Related to #
https://github.com/Automattic/dotcom-forge/issues/8808

## Proposed Changes

* The videopress iframe used the size. This PR calculates and calculates and sets the width and height based on the parent.

## Why are these changes being made?


* The videopress embeds were improperly shown in the help center

## Testing Instructions



* Use the live link and open the help center
* Search for "Preview" and open the relevant article
* Check if the video embed is shown properly
* Also check other articles containing embeds like: "Working With Video", "Add Links to Text", "Upload a Header Image", "Add Custom CSS"...

## Pre-merge Checklist
